### PR TITLE
refactor: migrate setting page to react component

### PR DESCRIPTION
**Pull request by ironAiken2**
### Feature 
- refactor SettingList and SettingItem components.
- use localstorage to remember the modifications made by the user. 

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by ironAiken2** [comment]
* **#2465** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2465?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ironAiken2 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/SettingList.tsx [hunk 1]
```diff
@@ -1,118 +1,96 @@
...
+            _.flatMap(settingGroup, (item) => item.settingItems).forEach(
+              (option) => {
+                option?.setValue && option.setValue(option.defaultValue);
+              },
+            );
+          }}
+        >
+          {t('button.Reset')}
+        </Button>
       </Flex>
```
Comment (yomybaby) [inline react/src/components/SettingList.tsx]: Please add a confirm popup for this feature.

File: react/src/components/SettingList.tsx [hunk 2]
```diff
@@ -1,118 +1,96 @@
...
       <Tabs
-        tabPosition={isLeftTab ? 'left' : 'top'}
-        tabBarStyle={{
-          width: isLeftTab ? leftTabWidth : undefined,
-        }}
-        items={appendedSettingGroup.map((settingGroup) => {
+        tabPosition={tabDirection ? 'left' : 'top'}
+        tabBarStyle={{ minWidth: 200 }}
+        items={_.map(appendedAllSettingGroup, (settingGroup) => {
```
Comment (yomybaby) [inline react/src/components/SettingList.tsx]: How about always adding `filterItems.length`? 
Please use secondary text for count.

**Review by yomybaby — CHANGES_REQUESTED** [review]
I haven't finished the review yet, but I'd like to leave some feedback first.

**Review by yomybaby — APPROVED** [review]
LGTM

**Pull request merged**